doc/topics/domain-postmortem.md

```diff
@@ -89,8 +89,8 @@ d.run(() => {
 ```
 
 It may be expected that nested domains always remain nested, and will always
-propagate the exception up the domain stack. Or that exceptions will never
-automatically bubble. Unfortunately both these situations occur, leading to
+propagate the exception up the domain stack. Or that exception will never
+automatically bubble. Unfortunately, both these situations occur, leading to
```

Comment:
A better change might to also add 'could' before 'occur' and 'of' before 'these.'